homeassistant/components/hydrawise/entity.py

```diff
@@ -3,31 +3,36 @@
 
 from typing import Any
 
+from homeassistant.helpers.device_registry import DeviceInfo
 from homeassistant.helpers.entity import EntityDescription
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
+from .const import DOMAIN, MANUFACTURER
 from .coordinator import HydrawiseDataUpdateCoordinator
 
 
 class HydrawiseEntity(CoordinatorEntity[HydrawiseDataUpdateCoordinator]):
```

Comment:
Shouldn't this entity get a unique id before this change?

Reply:
what do you mean?

Reply:
The prefix for the unique id is the device id, which is being added in this PR. I'm not sure how to add it in a separate PR?

Reply:
ooh lol, I think I didn't see the `_attr_unique_id` when I wrote this comment